Vic2ToHoI4/Data_Files/blankMod/output/events/NewsEvents.txt

```diff
@@ -1,3 +1,260 @@
+add_namespace = news
+# About Olympic games:
+# HoI4 syntax did not allow me to tell "check if major country is X and, if you find no major country
+# like X, then take a random minor"
+# Furthermore, the hidden_effect thing is needed to show the event
+# also to the player. It does not interfere with whoever is hosting
+# the Olympic games. If you're playing Switzerland and Australia hosts the games,
+# you'll read of Australian Olympics, not of Swiss ones.
+# Games of the XI Olympiad
+news_event = {
+	id = news.66 #Games of the XI Olympiad
+	title = news.66.t
+	desc = olympics.1936.d
+	picture = GFX_news_event_olympics_1940
+	
+	major = yes
+	
+	trigger = {
+		date > 1936.8.15
+		threat < 0.5
+		NOT = { has_global_flag = olympics_1936 }
+		OR = {
+			AND = {
```

Comment:
I'm not sure how I feel about this condition. Not all majors are necessarily faction leaders. Even if they are now (I can't recall), that's something liable to change in the future. Or does is_major mean something different in HoI4 terms?

Reply:
We ask the game for a major who is a faction leader and at peace. If it doesn't find any, a random minor is chosen.

Reply:
Could we have a middle option of majors who aren't faction leaders?

Reply:
That's for the following Olympic games. 1936 were held in Germany, who led the Axis, and 1940 were scheduled in Tokyo, who led no faction.

Reply:
I guess my concern is games that start with no factions. That means the majors get completely ignored in such a world.

Reply:
Fixed